lib/settings.js

```diff
@@ -51,20 +53,32 @@ function readFrom(envVars) {
         AMQP_PUBLISH_RETRY_ATTEMPTS: 10
     };
 
+    if (envVars.ELASTICIO_ADDITIONAL_VARS_FOR_HEADERS) {
+        const vars = {};
+        envVars.ELASTICIO_ADDITIONAL_VARS_FOR_HEADERS
+            .split(',')
+            .map(env => env.trim())
+            .forEach(env => {
+                const key = env.indexOf(PREFIX) === 0 ? env.slice(PREFIX.length) : env;
+                vars[key] = envVars[env];
```

Comment:
As far as i understand, at this moment envVars object contains key-value pairs
where key is prefixed. Here values are takes by key without prefix.
This seems to be strange. 
maybe tests works due to the fact that objects are passed by reference thought code?

Reply:
discussed in PM